cloudformation/service/eu-west-2/iam.yml

```diff
@@ -154,7 +154,7 @@ Resources:
                 Action:
                   - "s3:ListBucket"
                 Resource:
-                  - !Sub arn:aws:s3:::fdbt-csv-reference-data-${self:provider.stage}/*
+                  - !Sub arn:aws:s3:::fdbt-csv-reference-data-${self:provider.stage}
```

Comment:
`self:provider.stage` is a serverless variable, here it should just be Stage from the Parameters section

Reply:
Same for the other policy item

Reply:
Fixed!